modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt

```diff
@@ -286,6 +286,93 @@ class InstructionsTest {
         }
     }
 
+    @Test
+    @WithIroha
+    fun `burn and mint public key instruction committed`(): Unit = runBlocking {
+        // check Bob's public key before burn it
+        val bobPubKey = BOB_KEYPAIR.public.toIrohaPublicKey()
+        val query = QueryBuilder.findAccountById(BOB_ACCOUNT_ID)
+            .account(ALICE_ACCOUNT_ID)
+            .buildSigned(ALICE_KEYPAIR)
+        val signatories = client.sendQuery(query).signatories
+        assertEquals(1, signatories.size)
+        assertContentEquals(bobPubKey.payload, signatories.first().payload)
+
+        // burn Bob's public key
+        client.sendTransaction {
+            account(ALICE_ACCOUNT_ID)
+            burnPublicKey(BOB_ACCOUNT_ID, bobPubKey)
+            buildSigned(ALICE_KEYPAIR)
+        }.also {
+            Assertions.assertDoesNotThrow {
```

Comment:
I guess it's better to query Bob account again and check there is no such key present anymore. Though I assume it could be Iroha bug if that's possible for an account to exist without any key assigned

Reply:
agree, done